src/sync/sync_user.hpp

```diff
@@ -25,12 +25,27 @@
 #include <vector>
 #include <mutex>
 
+#include "../util/atomic_shared_ptr.hpp"
```

Comment:
`../` should be unnecessary as object store's `src` directory should be in the include path?